### PR TITLE
Updating leaderLogs.py to accept command line arguments

### PR DESCRIPTION
**Pull request by Crypto2099**
Updated the script to accept required command line arguments. Takes real values for all arguments except for vrf-skey which takes the JSON pool.vrf.skey file and parses it for compatibility.

### CODE REVIEW

File: leaderLogs/leaderLogs.py [hunk 1]
```diff
@@ -7,6 +7,29 @@
...
 
+import json
+import argparse
+
+parser = argparse.ArgumentParser(description="Calculate the leadership log.")
+parser.add_argument('--pool-id', dest='poolId', help='the pool ID', required=True)
+parser.add_argument('--epoch', dest='epoch', help='the epoch number [e.g. 221]', type=int, required=True)
+parser.add_argument('--epoch-nonce', dest='eta0', help='the epoch nonce to check', required=True)
+parser.add_argument('--vrf-skey', dest='skey', help='provide the path to the pool.vrf.skey file', required=True)
+parser.add_argument('--sigma', dest='sigma', type=float, help='the controlled stake sigma value of the pool', required=True)
```
Comment (papacarp) [inline leaderLogs/leaderLogs.py:18]: maybe give an example.  some people might submit 1% or something.  0.010052348379780869
`the controlled stake sigma value of the pool (example: 0.0034052348379780869) `

File: leaderLogs/leaderLogs.py [hunk 2]
```diff
@@ -21,12 +44,12 @@
 
 # more hard coded values
 local_tz = pytz.timezone('America/Los_angeles') # use your local timezone name here
-epoch=221
-poolId="95c4956f7a137f7fe9c72f2e831e6038744b6307d00143b2447e6443"
-sigma = 0.010052348379780869 # note function to pull data from ledger state is below.  its just faster to hard code it for testing
+#epoch=221
+#poolId="95c4956f7a137f7fe9c72f2e831e6038744b6307d00143b2447e6443"
+#sigma = 0.010052348379780869 # note function to pull data from ledger state is below.  its just faster to hard code it for testing
```
Comment (papacarp) [inline leaderLogs/leaderLogs.py:49]: feel free to just delete these lines as well

Comment (papacarp) [reply leaderLogs/leaderLogs.py:49]: the comments I mean with the old variable values

**Comment by Crypto2099** [comment]
sigma example has been added to the help text and old comments have been removed.

**Review by papacarp — APPROVED** [review]

**Pull request merged**